blackbox/docs/sql/scalar.txt

```diff
@@ -749,6 +749,19 @@ The second argument (``b``) is optional. If not present, base 10 is used::
     The same is true for arguments which lead to a ``division by zero``, as
     e.g. log(10, 1) does.
 
+power(a: number, b: number) returns double
+--------------------------------------------------
+Returns the given argument ``a`` raised to the power of argument ``b``.
+
+See below for an example:
+
+    cr> select power(2,3) from sys.cluster;
+    +---------------+
```

Comment:
Please fix output table. See: https://travis-ci.org/crate/crate/jobs/166565128#L1977
You can run these tests with: ./gradlew itest